trader/src/lib.rs

```diff
@@ -64,14 +83,14 @@ impl HookInitializer for TraderHook {
                                 Ok(v) => {
                                     let r =
                                         timeout(hook_data.task_timeout.clone(), v.execute()).await;
-                                    println!("{r:?}");
+                                    println!("Send trade result: {r:?}");
                                 }
                                 Err(e) => {
-                                    println!("{e:?}");
+                                    println!("Estimate Uniswap error: {e:?}");
                                 }
                             },
                             Err(e) => {
-                                println!("{e:?}");
+                                println!("Estimate mp swap error: {e:?}");
```

Comment:
Let's make it more into logging later, to be able to monitor on graphana, also we need a otel setup with metrics for our services